presto-product-tests/conf/docker/singlenode-mysql/docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
     command:
       mysqld --port 13306
     environment:
-      MYSQL_USER: swarm
-      MYSQL_PASSWORD: swarm
-      MYSQL_ROOT_PASSWORD: swarm
+      MYSQL_USER: test
+      MYSQL_PASSWORD: test
+      MYSQL_ROOT_PASSWORD: test
```

Comment:
i'd prefer passwords to be distinguishable. eg

```
user: test
password: mysqltestuserpassword
root password: mysqlrootuserpassword
```

Reply:
i won't insist though

Reply:
test/test is much easier to remember, just in case you would like to connect to mysql with mysql CLI